local.css

```diff
@@ -167,13 +167,24 @@ ol li li li {
     padding-left: .5em;
 	}
 
+.termlist th span:not(.hidden) {
+	display: inline-block;
+	}
 
+.termlist th {
+	min-width: 4em;
+    white-space: nowrap;
+	}
 
+.termlist {
+	border-collapse: collapse;
+	}
 
-
-
-
-
+.termlist th,
+.termlist td {
+	border: solid 1px #ccc,
```

Comment:
The comma should be a semicolon here.

Reply:
thank you for catching this! fixed by cb32fce